docs/DeveloperGuide.md

```diff
@@ -170,15 +170,48 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 This section describes some noteworthy details on how certain features are implemented.
 
 ### 4.1 Company detail panel (UI component)
-
 The `CompanyDetailPanel` allows user to view the company details of the selected company in the company list.
-The user can use the `view` command to select the company to view.
+Recruiter's information, company's information and remarks will be shown in the company detail panel.
 
 #### 4.1.1 Implementation
+`CompanyDetailCard` and `CompanyDetailPanel` both inheriting `UiPart` are used to display the company details. More details 
+of the class implementation can be seen in the class diagram below.
+
+<img src="images/DetailPanelClassDiagram.png" />
+
+In `CompanyDetailCard` the method `priorityFlowPane` is used to create a `FlowPane` to display the priority of the company.
+The color of the `FlowPane` is determined by the priority of the company. Red is used to indicate high priority,
+orange is used to indicate medium priority and green is used to indicate low priority. 
+
+As for the other information, FMXL labels are used to display the information. Within the constructor of `CompanyDetailCard`, 
+the respective FXML labels are set with the information of the company.
+
+To display the details in 3 different boxes, `CompanyDetailCard.fxml` is divided into 3 sections with each section 
+being a `VBox`. The 3 `VBox` are then added into a `HBox` to display the details in 3 different boxes.
+
+As for `CompanyDetailPanel`, there is an inner class `CompanyDetailViewCell` which extends `ListCell<Company>`. This class 
+setGraphics to the `CompanyDetailCard` by constructing a new `CompanyDetailCard` with the company details of the company.
```

Comment:
If you want to mention the method setGraphicsConfiguration, better to use "calls setGraphics ....". But if it doesn't exactly fit, then it's better to just explain in english: "sets the graphics"